### PR TITLE
Add homework 1 javascript

### PR DESCRIPTION
**Pull request by TatjanaOcedova**
Finished homework 1 javascript.

### CODE REVIEW

File: javascript/javascript1/week1/main.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+// Age-ify (A future age calculator)
+const yearOfBirth = 1986;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture + ".");
```
Comment (MLMeyer) [inline javascript/javascript1/week1/main.js:5]: Clean and concise code 👌
Great attention to detail making the yearOfBirth a const and adding a dot to the end of the log. 💫

File: javascript/javascript1/week1/main.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      humanYear +
+      " human years old in " +
+      dogYearFuture +
+      "."
+  );
+} else {
+  console.log(
+    "Your dog will be " + dogYear + " dog years old in " + dogYearFuture + "."
+  );
+}
```
Comment (MLMeyer) [inline javascript/javascript1/week1/main.js:25]: Great structure and once again good attention to detail. 👌

OBS! 😊
Have a second look at your boolean variable shouldShowResultInDogYears. What are you printing out if this value is true?

File: javascript/javascript1/week1/main.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    houseWide[i] * houseDeep[i] * houseHigh[i] * 2.5 * 1000 +
+    gardenSize[i] * 300;
+
+  if (housePrice > houseCosts[i]) {
+    console.log(names[i] + " is paying too little.");
+  } else {
+    console.log(names[i] + " is paying too much.");
+  }
+}
+
```
Comment (MLMeyer) [inline javascript/javascript1/week1/main.js:47]: Super clean structure and efficient code, well done! 🔥

File: javascript/javascript1/week1/main.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  "Staff",
+  "Agency",
+  "Crew",
+  "Team",
+];
+const startupName =
+  firstWords[Math.floor(Math.random() * 10)] +
+  secondWords[Math.floor(Math.random() * 10)];
+
+console.log("The startup: " + startupName + "!");
```
Comment (MLMeyer) [inline javascript/javascript1/week1/main.js]: Super well done! My startupName became "InnovativeAgency" - now I just need to come up with the service 😅

The assignment shows an example of the output being: "The startup: "Easy Corporation" contains 16 characters"

How could you update your string concatenation to get a similar result? 😊

Comment (TatjanaOcedova) [reply javascript/javascript1/week1/main.js]: I am very happy you like it, we have to start from somewhere even the name is a good start 😅
I should add startupName.length

File: javascript/javascript1/week1/main.js (hunk 2 above)
Comment (TatjanaOcedova) [reply javascript/javascript1/week1/main.js:25]: I have mixed up the true and false statements. I will make it to be true and show the result in dog years.

**Review by MLMeyer — APPROVED** [review]

**Pull request merged**